hyperspy/_signals/eds_tem.py

```diff
@@ -452,3 +452,30 @@ def decomposition(self,
             navigation_mask=navigation_mask, *args, **kwargs)
         self.learning_results.loadings = np.nan_to_num(
             self.learning_results.loadings)
+
+    def create_model(self, auto_background=True, auto_add_lines=True,
+                     *args, **kwargs):
+        """Create a model for the current TEM EDS data.
+
+        Parameters
+        ----------
+        auto_background : boolean, default True
+            If True, adds automatically a polynomial order 6 to the model, using
+            the edsmodel.add_polynomial_background method.
+        auto_add_lines : boolean, default True
+            If True, automatically add Gaussians for all X-rays generated in the
+            energy range by an element using the edsmodel.add_family_lines
+            method.
+
+        Returns
+        -------
+
+        model : `EDSTEMModel` instance.
+
+        """
```

Comment:
As this replaces the docstring of the underlying method, the parameters of the general model should also be listed here. I don't know if there is an automated way to add that (doubt it), so double typing is probably necessary (copy-paste from Model docstring / deprecated `hspy.create_model()`)